### PR TITLE
Add Window type to JSDoc validator

### LINKED ISSUE #250: JSDoc validator: Add Window as a valid type
ATM there's no way to document `@param {Window}` because it throws `Incorrect param type: Window`. It should be added [here](https://github.com/ckeditor/ckeditor5-dev/blob/master/packages/jsdoc-plugins/lib/validator/types.js).

A follow-up of https://github.com/ckeditor/ckeditor5-utils/pull/174.

### PR DESCRIPTION
**Pull request by ma2ciek**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Internal: Added Window type to JSDoc validator. Closes #250.

### CODE REVIEW

File: packages/jsdoc-plugins/lib/validator/types.js [hunk 1]
```diff
@@ -35,6 +35,7 @@ const BASIC_TYPES = [
 	'ClientRect',
 
 	// Web APIs
```
Comment (Reinmar) [inline packages/jsdoc-plugins/lib/validator/types.js]: Ekhm... shouldn't it belong to the previous category?

Comment (ma2ciek) [inline packages/jsdoc-plugins/lib/validator/types.js]: Hmm, but that's something more than just DOM. I should create another category or write it at the top level.

Comment (Reinmar) [inline packages/jsdoc-plugins/lib/validator/types.js]: Window is part of DOM: https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model

Comment (ma2ciek) [inline packages/jsdoc-plugins/lib/validator/types.js]: I would say that it implements some DOM interfaces, but not belongs to, as it contains huge amount of other things

**Pull request closed without merging**